libpod/image/image.go

```diff
@@ -881,6 +887,7 @@ func (i *Image) Inspect(ctx context.Context) (*inspect.ImageData, error) {
 		User:         ociv1Img.Config.User,
 		History:      ociv1Img.History,
 	}
+	defer i.newImageEvent(events.Inspect)
```

Comment:
Again, I really don't think we need an event for this

Reply:
Ditto